phoenix5-spark3/README.md

```diff
@@ -39,7 +80,9 @@ UPSERT INTO TABLE1 (ID, COL1) VALUES (2, 'test_row_2');
 ```
 
 ### Load as a DataFrame using the DataSourceV2 API
+
 Scala example:
+
```

Comment:
Maybe add comments to make  it obvious that you need to use a real ZK quorum , like 
`//replace "phoenix-server:2181" with the real ZK quorum`